src/main/kotlin/org/malachite/estella/interview/api/InterviewController.kt

```diff
@@ -23,7 +23,7 @@ class InterviewController(
     fun getJobseekerName(@PathVariable interviewId: InterviewId): ResponseEntity<JobseekerName> =
         interviewService.getUserFromInterviewUuid(interviewId.toUUID())
                 .let {
-                    ResponseEntity.ok(JobseekerName(it!!.firstName, it.lastName))
+                    ResponseEntity.ok(JobseekerName(it?.firstName, it?.lastName))
```

Comment:
Tak w sumie co to zwróci jak it nie będzie miało tych Danych?

Reply:
Zasadniczo nic, jest to odpowiedz na propozycje Dawida żeby nie rzucać wyjątkiem jeśli nie trzeba, i ze lepiej mieć część imienia niż błąd, kosztem tego jest fakt ze możemy mieć nic xD

Reply:
Znaczy no wg mnie to będzie dosyć dziwne. Chyba że zawsze się na froncie filtruje to co przyjdzie z backu i sprawdza czy to nie jest nullem XD @Davenury